src/java/knet/src/main/java/org/mases/knet/connect/KNetConnectProxy.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2022 MASES s.r.l.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Refer to LICENSE for more information.
+ */
+
+package org.mases.knet.connect;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+import org.mases.jcobridge.*;
+import org.mases.knet.connect.source.KNetSourceTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class KNetConnectProxy {
+    private static final Logger log = LoggerFactory.getLogger(KNetSourceTask.class);
+
+    public static final String DOTNET_CLASSNAME_CONFIG = "classname";
```

Comment:
Convert the property name from **classname** to **knet.classname**: in this way the specific properties of KNet will have a common prefix